### PR TITLE
Finished Assignment# 3

### PR DESCRIPTION
**Pull request by DemonHiatus**

### CODE REVIEW

File: src/A3Q1.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
                 rem.move();
                 
             } 
-            
+            // || means or/and
```
Comment (DemonHiatus) [inline src/A3Q1.java]: Fixed unnecessary comment.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/nbproject/private/
```
Comment (DemonHiatus) [inline .gitignore:1]: No Idea what this is for.

Comment (MrLamont-HHSS) [reply .gitignore:1]: GitIgnore is a file which states which files shouldn't be uploaded when you do a push to GitHub. The review function is for when you are working in groups or looking at someone elses code and you'd like to make a comment about a specific file from their code.

**Comment by MrLamont-HHSS** [comment]
Excellent work on the programs. Some of the comments are a little vague and could be expanded on.

**Pull request closed without merging**